interface/src/app/tribe_client/geneset-result-table.tpl.html

```diff
@@ -0,0 +1,14 @@
+<div>
+    <table class="table table-striped" name="searchResults">
+        <tr><th>Title</th><th>Species</th><th>Creator</th></tr>
```

Comment:
Minor quibble: if the table header were formatted the same as the table rows below, it would be easier to see which heading goes with which column.